### PR TITLE
Implemented a DELETE method for the comments table in the nc_news dat…

### PR DESCRIPTION
**Pull request by AlfarooqAlmenghawi**
…abase, allowing the client to delete a comment from the database.

### CODE REVIEW

File: MVC/controllers/comments.controller.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+const deleteSpecificComment = (request, response, next) => {
+  const { comment_id } = request.params;
+
+  return removeSpecificComment(comment_id)
+    .then(() => {
+      response.status(204).send(); // Nothing can be sent, it will always assume nothing
+    })
+    .catch((error) => {
+      next(error);
```
Comment (nc-jodie-cheung) [inline MVC/controllers/comments.controller.js:32]: Please remove your console logs 👀

File: MVC/models/comments.model.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        return Promise.reject(nonExistentCommentCustomError);
+      }
+      return result;
+    });
+};
+
+const removeSpecificComment = (comment_id) => {
+  return fetchSpecificComment(comment_id).then((result) => {
+    return db.query(`DELETE FROM comments WHERE comment_id = $1`, [comment_id]);
+  });
```
Comment (nc-jodie-cheung) [inline MVC/models/comments.model.js:27]: You don't need to invoke `fetchSpecificComment` in order to delete it. You can handle it directly by deleting the comment using its comment_id 💡

File: __tests__/app.test.js [hunk 3]
```diff
@@ -465,6 +532,34 @@ describe("PATCH", () => {
...
+
+  test("/api/comments/:comment_id 410: should reject and return a 406 indicating that the parameter is not a nuber (ID).", () => {
+    return request(app)
+      .delete("/api/comments/HAVGVDG")
+      .expect(406)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          message: "Unknown endpoints are not acceptable on this platform.",
+        });
+      });
```
Comment (nc-jodie-cheung) [inline __tests__/app.test.js:559]: Please refactor your tests here. You would need to account for the following:

- Status 404, non existent ID, e.g 999
- Status 400, invalid ID, e.g "not-an-id"

Try to keep your test descriptions clear and concise 💡

**Review by nc-jodie-cheung — COMMENTED** [review]
Please see comments 😄

File: MVC/models/comments.model.js (hunk 2 above)
Comment (AlfarooqAlmenghawi) [reply MVC/models/comments.model.js:27]: I know but I needed to throw an error if the comment doesn't exist in the first place, or is that unnecessary

File: __tests__/app.test.js (hunk 3 above)
Comment (AlfarooqAlmenghawi) [reply __tests__/app.test.js:559]: I did, from line 441 to line 473

**Pull request merged**